ui/store/actions.ts

```diff
@@ -5504,6 +5486,32 @@ export function fetchAndUpdateMetamaskNotifications(
   };
 }
 
+/**
+ * Deletes notifications by their id.
+ *
+ * This function sends a request to the background script to delete notifications by the passed in ids and updates the state accordingly.
+ * If the operation encounters an error, it logs the error message and rethrows the error to ensure it is handled appropriately.
+ *
+ * @param ids - The ids of the notifications to delete.
+ * @returns A thunk action that, when dispatched, attempts to delete a notification by its id.
+ */
+export function deleteNotificationsById(
+  ids: string[],
+): ThunkAction<void, MetaMaskReduxState, unknown, AnyAction> {
+  return async () => {
+    try {
+      const response = await submitRequestToBackground(
+        'deleteNotificationsById',
+        [ids],
```

Comment:
Is it intentional to wrap an array in another array?

Reply:
I think that's how the function accepts params, `deleteNotificationsById` expects an array.

Reply:
Ah yes, that makes sense! I hadn't realized that `submitRequestToBackground` requires params to be an array of params.